content/en/logs/guide/aws-eks-fargate-logs-with-kinesis-data-firehose.md

```diff
@@ -0,0 +1,178 @@
+---
+title: AWS EKS Fargate logs with Kinesis Data Firehose
+kind: guide
+further_reading:
+- link: https://docs.datadoghq.com/logs/log_configuration/processors/
+  tag: Documentation
+  text: Processors 
+- link: https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html
+  tag: Documentation
+  text: Fargate logging
+- link: https://docs.aws.amazon.com/eks/latest/userguide/fargate-profile.html
+  tag: Documentation
+  text: AWS Fargate profile
+---
+
+## Overview
+
+AWS Fargate on EKS provides a fully managed experience for running Kubernetes workloads. Kinesis Data Firehose can be used with EKS's Fluentbit log router to collect logs in Datadog.
+
+{{< img src="logs/guide/aws-eks-fargate-logs-with-kinesis-data-firehose/log_streaming_diagram.png" alt="Diagram of the log flow depicting a Fargate EKS cluster sending container logs through Fluentbit log router to Kinesis data firehose and an S3 backup bucket within AWS and then on to Datadog" responsive="true">}}
+ 
+   
+### Kinesis Data Firehose vs Cloudwatch log forwarding {#firehose-vs-cloudwatch}
```

Comment:
```suggestion
### Kinesis Data Firehose and CloudWatch log forwarding
```

Reply:
I'm not sure what the  `{#firehose-vs-cloudwatch}` does in Markdown, but as far as I can tell this doesn't seem to have any effect. Let me know if there's something I'm missing here!